tsl/test/sql/continuous_aggs.sql.in

```diff
@@ -1490,8 +1490,11 @@ SELECT time_bucket('1day', timec), min(location), sum(temperature), sum(humidity
 FROM conditions
 GROUP BY time_bucket('1day', timec) WITH NO DATA;
 
+-- enable partial aggregates only for postgres version <= 14
+SELECT CASE WHEN current_setting('server_version_num')::int/10000 <= 14 THEN 'false' ELSE 'true' END AS "final_flag" \gset
+
```

Comment:
Same as above!

Reply:
I thought we need 1 -ve tests case which runs on all PG version. Thus made this change.